GDevelop.js/__tests__/MetadataDeclarationHelper.js

```diff
@@ -1099,6 +1113,53 @@ describe('MetadataDeclarationHelper', () => {
     project.delete();
   });
 
+  it('can create metadata for custom object with all capabilities', () => {
+    const extension = new gd.PlatformExtension();
+    const project = new gd.Project();
+
+    const eventExtension = project.insertNewEventsFunctionsExtension(
+      'MyExtension',
+      0
+    );
+    const eventsBasedObject = eventExtension
+      .getEventsBasedObjects()
+      .insertNew('MyObject', 0);
+    eventsBasedObject.markAsRenderedIn3D(true);
+    // TODO Uncomment
```

Comment:
Be more precise in your TODO, otherwise if I grep the codebase search for "TODO", I have no information about 1) is this a real TODO or not 2) is this urgent 3) is this dangerous.